index.rst

```diff
@@ -7,6 +7,7 @@ Welcome to the ros2_control documentation!
    :hidden:
 
    getting_started.rst
+   ros2_controllers/doc/controllers_index.rst
```

Comment:
I propose to  use just `index.rst` file

```suggestion
   ros2_controllers/doc/index.rst
```

Reply:
It's much easier to find files when they are not called the same way.

Reply:
Also note that this isn't necessarily The Entrypoint. It's merely collecting the userdoc files.  A natural extension of this controller-repo inclusion is to run doxygen too, which should probably be a separate page, or perhaps grouped per-controller with the userdoc. We can work that out later.